From 6ee66a4736bde681956ef8ab601a72a5cb7b19ed Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 20 Jan 2023 18:30:06 +0100
Subject: [PATCH] bootctl: change section title for kernel image commands

Let's call them kernel *images*, not just *kernels*.

(cherry picked from commit 1e7d6cc07211de425bcc5c408c1f4376d6717305)

Related: RHEL-16354
---
 man/bootctl.xml    | 2 +-
 src/boot/bootctl.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/man/bootctl.xml b/man/bootctl.xml
index c12fe93214..d82f12d5bb 100644
--- a/man/bootctl.xml
+++ b/man/bootctl.xml
@@ -218,7 +218,7 @@
   </refsect1>
 
   <refsect1>
-    <title><command>kernel</command> Commands</title>
+    <title>Kernel Image Commands</title>
 
     <variablelist>
       <varlistentry>
diff --git a/src/boot/bootctl.c b/src/boot/bootctl.c
index 681c5bd44f..4f2a6288fb 100644
--- a/src/boot/bootctl.c
+++ b/src/boot/bootctl.c
@@ -1454,9 +1454,9 @@ static int help(int argc, char *argv[], void *userdata) {
                "  remove              Remove systemd-boot from the ESP and EFI variables\n"
                "  is-installed        Test whether systemd-boot is installed in the ESP\n"
                "  random-seed         Initialize random seed in ESP and EFI variables\n"
-               "\n%3$skernel Commands:%4$s\n"
+               "\n%3$sKernel Image Commands:%4$s\n"
                "  kernel-identify     Identify kernel image type\n"
-               "  kernel-inspect      Prints details about the kernel\n"
+               "  kernel-inspect      Prints details about the kernel image\n"
                "\n%3$sOptions:%4$s\n"
                "  -h --help            Show this help\n"
                "     --version         Print version\n"
